chef/cookbooks/monasca/attributes/default.rb

```diff
@@ -41,3 +41,8 @@
 default[:monasca][:log_agent][:user] = "monasca-log-agent"
 default[:monasca][:log_agent][:group] = "logstash"
 default[:monasca][:log_agent][:debug] = monasca_debug
+
+default[:monasca][:ha][:enabled] = false
+# Ports to bind to when haproxy is used for the real ports
+default[:monasca][:ha][:ports][:plain] = 5580
```

Comment:
This is just a 'random' port, I only wanted to experiment with HA to make sure I get it right. So can I assume that, in overall, the approach itself is correct?